### PR TITLE
Update nvm-tree API, layout change, and adhere to users shell

### PR DESCRIPTION
**Pull request by J-C-3**
nvime-tree for `update_to_buf_dir` has changed to `hijack_directories`

https://github.com/kyazdani42/nvim-tree.lua/blob/0c13bd76a897390a43e262d5716d5faa8367b8d8/doc/nvim-tree-lua.txt#L412

### CODE REVIEW

**Comment by J-C-3** [comment]
Update so `vimterm` uses the users shell and not specifically `zsh`

File: lua/globals.lua [hunk 1]
```diff
@@ -31,7 +31,7 @@ vim.o.pumblend = 15
 vim.o.relativenumber = true
 vim.o.ruler = true
 vim.o.scrolloff = 2
-vim.o.shell = "/usr/bin/env zsh"
+vim.o.shell = "/usr/bin/env sh"
```
Comment (distek) [inline lua/globals.lua:34]: This won't always work, should we just use `os.getenv("SHELL")` instead?

Comment (distek) [reply lua/globals.lua:34]:
```
╰(busypanini)─(INS)─▶ echo $SHELL
/bin/zsh
╭(quark)─(0)─()─(~)
╰(busypanini)─(INS)─▶ /usr/bin/env sh
sh-5.1$ 
```

**Comment by J-C-3** [comment]
Quashed commits and switched to `os.getenv`

File: lua/globals.lua (hunk 1 above)
Comment (J-C-3) [reply lua/globals.lua]: :+1:

**Comment by distek** [comment]
Doap Certified. Thanks!

**Pull request merged**